package com.mindtree.Exception;



public class Main {
	public void x() throws MyException {
		
		throw new ArithmeticException();
	}
	
	public void y() throws MyException {
		
		try{
			x();
			
		}catch(MyException e){
			MyException(e.getMessage(), e.getCause());
		}
		
	}
	public void z() {
		try {
			
		}
	}

	public static void main(String[] args) {
		

	}

}
